### PR TITLE
Use setTimeout hack (again) to get paste context in token search

### LINKED ISSUE #1457: Copy/Paste functionality not working on "Search Tokens" input field
**Describe the bug**
When you're on the Add Asset view, and attempt to paste text into the "Search Tokens" input field, you are unable to 

**Screenshots**
Seen here = https://recordit.co/j65X7HRTPn

**To Reproduce**
_Steps to reproduce the behavior_
1. Go to Wallet
2. Tap on + ADD TOKENS button
3. Attempt to paste into the "Search Tokens" input field

**Expected behavior**
_A clear and concise description of what you expected to happen_

User should be able to paste text into this field 

**Smartphone (please complete the following information):**

Seems to only be happening on Android

 - Device: both Google Pixel 2 and 3 and Samsung Note 9
 - OS: Android Q [version 10]
 - Version [Android 29 MM 0.2.14(27)]

-------------------------------------------------------------
_to be added after bug submission by internal support / PM_
**Severity**

- How critical is the impact of this bug on a user?
- Add stats if available on % of customers impacted
- Is this visible to all users?
- Is this tech debt?

### PR DESCRIPTION
**Pull request by rickycodes**
**Description**

re: #1457 Oddly this didn't seem to work the last time I tried it, but it does seem to do the trick now?

Additionally this only seems to work for paste, once there is text in the field it doesn't appear that it can be selected and copied, but that seems less important? It might make sense to create a component that accomplishes this application wide since we're doing it in so many places now.

**Checklist**

* [x] There is a related GitHub issue
* [x] Tests are included if applicable
* [x] Any added code is fully documented

**Issue**

Resolves #1457

### CODE REVIEW

File: app/components/UI/AssetSearch/index.js [hunk 1]
```diff
@@ -60,6 +60,10 @@ export default class AssetSearch extends PureComponent {
 		onSearch: PropTypes.func
 	};
 
+	componentDidMount() {
+		setTimeout(() => this.setState({ inputWidth: '86%' }), 100);
```
Comment (estebanmino) [inline app/components/UI/AssetSearch/index.js:64]: It wasn't 99% to 100% as all the other fixes?

Comment (rickycodes) [reply app/components/UI/AssetSearch/index.js:64]: I think the issue is the width just needs to be updated for it to work. In this case changing it from 99% to 100% pushed the magnifying glass out of the text box. Using these values brings the paste context back while ensuring things look proper.

Comment (estebanmino) [reply app/components/UI/AssetSearch/index.js:64]: cool makes sense

**Review by estebanmino — APPROVED** [review]

**Review by ibrahimtaveras00 — APPROVED** [review]
Fix looks good on Android and still works on iOS 👍

**Pull request merged**